pkg/plugins/platform/local.go

```diff
@@ -498,3 +498,8 @@ func (l *LocalApp) AddHostsEntry() error {
 	err = system.RunCommandPipe("sudo", hostnameArgs)
 	return err
 }
+
+// constructContainerName builds a container name given the type (web/db/dba) and the appName
+func constructContainerName(containerType string, appName string) string {
```

Comment:
Nice. This is actually just a simplification of the inline sprintf that was there previously. Sounds like your approach would be excellent.